injected/integration-test/test-pages/shared/utils.js

```diff
@@ -81,7 +81,7 @@ async function renderResults() {
     }
     // @ts-expect-error - ongoingTests is not defined in the type definition
     for (const test of window.ongoingTests) {
-        const result = await test.test();
+        const result = await test.test().catch((e) => console.error(`${test.name} threw`, e));
```

Comment:
exceptions from the in-page test runner were being swallowed, this just exposes them until there's a more advances runner.